framework_tvb/setup.py

```diff
@@ -40,15 +40,15 @@
 import shutil
 import setuptools
 
-VERSION = "2.0.9"
+VERSION = "2.0.10"
 
 TVB_TEAM = "Mihai Andrei, Lia Domide, Stuart Knock, Bogdan Neacsa, Paula Popa, Paula Sansz Leon, Marmaduke Woodman"
 
-TVB_INSTALL_REQUIREMENTS = ["allensdk", "BeautifulSoup4", "cherrypy", "cryptography", "flask", "flask-restplus",
-                            "formencode", "gevent", "h5py", "Jinja2", "networkx", "nibabel", "numpy", "Pillow",
-                            "psutil", "pyAesCrypt", "python-keycloak", "pyunicore",
-                            "scikit-learn", "scipy", "simplejson", "sqlalchemy", "sqlalchemy-migrate",
-                            "tvb-data", "tvb-gdist", "tvb-library"]
+TVB_INSTALL_REQUIREMENTS = ["allensdk", "cherrypy", "cryptography", "flask", "flask-restplus", "formencode",
+                            "gevent", "h5py", "Jinja2", "nibabel", "numpy", "pandas", "Pillow",
+                            "psutil", "pyAesCrypt", "python-keycloak", "pyunicore", "requests", "scikit-learn",
+                            "scipy", "simplejson", "six", "sqlalchemy", "sqlalchemy-migrate",
+                            "tvb-data", "tvb-gdist", "tvb-library", "werkzeug"]
 
```

Comment:
These changes are also included in PR #245. Maybe some rebasing error?

Reply:
Yes. We needed most of these in #245 also.
But I prefer to merge this PR first and fix the conflict there afterwards.